### ISSUE TITLE
16k EQ Slider is Reversed

### ISSUE THREAD

**Issue by Catdumpling** [clementine-player/Clementine#5564]
### System information

Please provide information about your system and the version of Clementine used.

- Operating System: Windows 7 Pro 64-bit (all OS updates current as of Dec. 22, 2016.)
- Clementine version: 1.3.1

### Expected behaviour / actual behaviour: 

16k EQ slider should increase frequency when moved up and decrease it when pulled down. 16k EQ slider is reversed (moving slider up decreases freq, lowering slider increases it.)

### Steps to reproduce the problem (only for bugs):

Adjust 16k EQ slider.

**Comment by santigl** [comment]
Hi, @Catdumpling. This should be already fixed. Could you try installing [this](http://builds.clementine-player.org/win32/release/ClementineSetup-1.3.1-267-g265f27a.exe) latest build and checking it again?

**Comment by volleimer** [comment]
Hi, has this issue been fixed? I am currently on Arch Linux, Clementine 1.3.1, build date 2018-05-02. The slider is still reversed here.

**Comment by santigl** [comment]
Yep, @volleimer, it was fixed, but unfortunately the changes were posterior to the release of 1.3.1.
